src/main/java/net/dries007/tfc/client/overworld/LevelRendererExtension.java

```diff
@@ -402,6 +406,20 @@ public boolean renderSnowAndRain(ClientLevel level, int ticks, float partialTick
 
             int stateFlag = -1;
 
+            Vec2 wind = ClimateRenderCache.INSTANCE.getWind();
+
+            // max angle is at ~50 kmh
+            float zAngleRain = Mth.TWO_PI / (360f / Mth.clampedMap(wind.x, 0, 0.4f, 0, RAIN_MAX_ANGLE));
+            float xAngleRain = Mth.TWO_PI / (360f / Mth.clampedMap(wind.y, 0, 0.4f, 0, RAIN_MAX_ANGLE));
+            float xAngleSnow = Mth.TWO_PI / (360f / Mth.clampedMap(wind.x, 0, 0.4f, 0, SNOW_MAX_ANGLE));
+            float zAngleSnow = Mth.TWO_PI / (360f / Mth.clampedMap(wind.y, 0, 0.4f, 0, SNOW_MAX_ANGLE));
+
```

Comment:
`final` ?

Reply:
Also I like to use `Constants#DEG_TO_RAD` and `Constants#RAD_TO_DEG` but i'm not a stickler about it